### ISSUE TITLE
word with tables connot replace content

### ISSUE THREAD

**Issue by qweawdqq** [unidoc/unioffice#267]
## Description
I want to replace some words in word. I find that if there is a table in word, the words in the table can not be replaced. Is there a solution?

## Expected Behavior
Normally replace labels in word

## Actual Behavior
As long as there are tables in the template, the contents of the tables can not be recognized and replaced.

### Example 
The following column is the content of word

## word title
this is word

abcd | 【aj∶axax】 | bbb | 【aj∶bxbx】
-- | -- | -- | --

word text bala bala balabala bala balabala bala balabala bala bala

Think about what's in word.
I need to replace [aj: axaxax] with [thank: you], because the label that needs to be replaced is in the table, so it can not be replaced, and the content outside the table can be replaced normally.

> func OutPutWord(inputFileName, outputFileName, ajlx string) {
	doc, err := document.Open(inputFileName)
	if err != nil {
		log.Fatalf("error opening document: %s", err)
	}
	paragraphs := []document.Paragraph{}
	for _, p := range doc.Paragraphs() {
		paragraphs = append(paragraphs, p)
	}
	for _, sdt := range doc.StructuredDocumentTags() {
		for _, p := range sdt.Paragraphs() {
			paragraphs = append(paragraphs, p)
		}
	}
	for _, p := range paragraphs {
		for _, r := range p.Runs() {
			switch r.Text() {
			case "«":
				r.ClearContent()
			case "»":
				r.ClearContent()
			default:
				r.ClearContent()
		                r.AddText(text)
			}
		}
	}
	doc.SaveToFile(outputFileName)
}

**Comment by nkryuchkov** [comment]
Hi, @qweawdqq . I do not see an issue here. To replace strings in the table contents you should loop through the doc's tables, rows and cells, like this:

```
	for _, t := range doc.Tables() {
		for _, row := range t.Rows() {
			for _, cell := range row.Cells() {
				paragraphs = append(paragraphs, cell.Paragraphs()...)
			}
		}
	}
```

**Comment by nkryuchkov** [comment]
Hi, @qweawdqq . After a discussion changes were made to the underlying func, changes are published within the #280 . With these changes your code should work as it is
